meson.build

```diff
@@ -102,41 +100,25 @@ wayland_client = dependency('wayland-client')
 wayland_egl    = dependency('wayland-egl')
 wayland_protos = dependency('wayland-protocols', version: '>=1.17')
 egl            = dependency('egl')
-freerdp        = dependency('freerdp2', required: get_option('freerdp'))
-winpr2         = dependency('winpr2', required: get_option('freerdp'))
 glesv2         = dependency('glesv2')
 drm            = dependency('libdrm', version: '>=2.4.95')
 gbm            = dependency('gbm', version: '>=17.1.0')
 libinput       = dependency('libinput', version: '>=1.9.0')
 xkbcommon      = dependency('xkbcommon')
 udev           = dependency('libudev')
 pixman         = dependency('pixman-1')
-libcap         = dependency('libcap', required: get_option('libcap'))
-logind         = dependency('lib' + get_option('logind-provider'), required: get_option('logind'), version: '>=237')
 math           = cc.find_library('m')
 rt             = cc.find_library('rt')
```

Comment:
Minor style nit: while at it, can we stop cargo-culting the dependencies alignment here? We don't align in C, so I don't think we should align in Meson files.

Reply:
Done